fastlane/jetpack_metadata/he/description.txt

```diff
@@ -0,0 +1,9 @@
+להשתמש בכלים עוצמתיים לאבטחה ולביצועים ישירות מהטלפון עם Jetpack ל-iOS.
+
+לשחזר את האתר מכל מקום אם משהו משתבש. לסרוק את האתר לאיתור איומים ולפתור אותם בהקשה אחת. לעקוב אחר הפעילות באתר כדי לראות מי שינה מה ומתי. לבדוק את הנתונים הסטטיסטיים שלך כדי לראות מאיזו מדינה נכנסו היום המבקרים באתר שלך.
+
+האפליקציה תומכת גם באפשרויות המוכרות של WordPress. לשבת על הספה ולכתוב חמשיר. לצלם תמונה בהפסקת הצהריים ולפרסם אותה. לענות על התגובות האחרונות שקיבלת.
+
+האפליקציה של Jetpack ל-iOS תומכת באתרי WordPress.com שבהם מופעל השירות של Jetpack ושאליהם מחובר חשבון WordPress.com.
+
+ניתן לעיין בהצהרת הפרטיות למשתמשים מקליפורניה בכתובת https://automattic.com/privacy/#california-consumer-privacy-act-ccpa.
```

Comment:
This is sort of the opposite of the comment above regarding deleted release notes.

#16764 added back the Hebrew locale, resulting in all the already translated metadata (notice there's no 17.7 release notes) being downloaded this time around.